dev/VSIX/Directory.Build.targets

```diff
@@ -77,7 +77,7 @@
                 <OriginalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId)</OriginalNugetRepositoryId>
                 <!-- 'ExperimentalNugetRepositoryId' needs to match the value set by the XmlPoke query for
                 "/ns:PackageManifest/ns:Metadata/ns:Identity/@Id" in Extension\WindowsAppSDK.Extension.csproj -->
-                <ExperimentalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId).Experimental</ExperimentalNugetRepositoryId>
+                <ExperimentalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId).Preview</ExperimentalNugetRepositoryId>
```

Comment:
Does this make it so the name of the vsix file now has preview in it instead of experimental?

Reply:
I thought that VS used the VSIX Id, not the filename. I'm going to try testing it now.

Reply:
It needs to match the VSIX ID, as I noted in the comment. :)